webapp/src/components/Footer/index.js

```diff
@@ -26,10 +26,10 @@ const Footer = () => {
       </div>
       <div className={classes.footerInformation}>
         <p>EOS Costa Rica</p>
-        <a href="https://es.eoscostarica.io/">Website EOS Costa Rica</a>
+        <a href="https://edenia.com/es">Website EDENIA Costa Rica</a>
```

Comment:
Why is this linking to spanish?

Website EDENIA Costa Rica ?

lets replace this with: `Edenia.com  Website` ?